index.css

```diff
@@ -14,7 +14,60 @@
   box-sizing: border-box;
   font-family: "Roboto", sans-serif;
 }
+/*Header Styles */
+header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.nav{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.logo {

```

Comment:
You don't need to change it, but you can use a shorthand for margin.
If you have four different values, you can write it like this:
`margin: <top> <right> <bottom> <left>`
Notice there are no commas.
So in this case you could write `margin: 10px 40px 10px 10px`
If your right and left margin are the same, but top and bottom are different you can write it like this:
`margin: <top> <right and left> <bottom>`
If top and bottom are the same value and right and left are the same value (different from top and bottom), the shorthand uses two values:
`margin: <top and bottom> <right and left>`
And if the margin is the same on all four sides, you just define margin with that one value.